validate_sptx/test_fuzz.py

```diff
@@ -0,0 +1,79 @@
+import os
+from pkg_resources import resource_filename
+
+from .util import SpaceTxValidator
+from .util import Fuzzer
+
+codebook_schema_path = resource_filename("validate_sptx", "schema/codebook/codebook.json")
+experiment_schema_path = resource_filename("validate_sptx", "schema/experiment.json")
+codebook_validator = SpaceTxValidator(codebook_schema_path)
+experiment_validator = SpaceTxValidator(experiment_schema_path)
+
+
+def test_fuzz_mock():
```

Comment:
Please add some docstrings to the tests in this class explaining: 
1. The fixtures
2. What the testing strategy is.